doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -278,6 +279,12 @@ subscription creation/update time. Check details in the document about [initial
 
 [Top](#top)
 
+## Oneshot subscriptions
+
+Apart from the `status` values defined for subscription in the NGSIv2, Orion also allows to use `oneshot`. Please find details in this [oneshot_subscription.md](oneshot_subscription.md)
```

Comment:
NGSIv2 -> NGSIv2 specification

`[oneshot_subscription.md](oneshot_subscription.md)` -> `[piece of documentation](oneshot_subscription.md)`

Reply:
fixed in 5c2a50e